Gems/Atom/Utils/Code/Include/Atom/Utils/ImGuiGpuProfiler.inl

```diff
@@ -834,8 +834,10 @@ namespace AZ
                             {
                                 // Check whether it should be sorted by name.
                                 const uint32_t sortType = static_cast<uint32_t>(m_sortType);
+                                AZ_PUSH_DISABLE_WARNING(4296, "-Wunknown-warning-option")
                                 bool sortByName = (sortType >= static_cast<uint32_t>(ProfilerSortType::Alphabetical) &&
                                     (sortType < static_cast<uint32_t>(ProfilerSortType::AlphabeticalCount)));
+                                AZ_POP_DISABLE_WARNING
```

Comment:
Is MSVC incorrectly warning about this these boolean statements comparing values against an enum being always true?

Reply:
the issue here is that `sortType` is unsigned and `ProfilerSortType::Alphabetical` is 0, so it will always be true
However, I am wrapping it because `ProfilerSortType::Alphabetical` could shift in the enum and stop being `0`
Something similar is happening with a bunch of code from Multiplayer, they have empty enums that will fill at some point